.clang-format

```diff
@@ -0,0 +1,11 @@
+BasedOnStyle: Google
+AlignAfterOpenBracket: AlwaysBreak
```

Comment:
@dstockwell I just approved, but I remembered a question. How'd you arrive at this list of exceptions?

Reply:
That's a good question. They were sourced internally but removing them, the only one that has any affect is ReflowComments, so they are probably all part of the google config now. I'll reset to just `BasedOnStyle: Google`